qsiprep/workflows/dwi/finalize.py

```diff
@@ -343,6 +343,7 @@ def init_dwi_finalize_wf(
     series_qc = pe.Node(SeriesQC(output_file_name=output_prefix), name="series_qc")
     ds_series_qc = pe.Node(
         DerivativesDataSink(
+            space="ACPC",
             desc="ImageQC",
```

Comment:
I wonder if it makes sense to have desc-ImageQC, when it has the stats for desc-preproc_dwi. wdyt?

Reply:
I don't think the `desc` entity needs to match up, but at least in XCP-D I have `desc-linc_qc` instead of `desc-LINCQC_bold`. So what about `desc-image_qc` and `desc-slice_qc`?

Reply:
I like that much more. The capitalization of the previous name is also very non-bidsy

Reply:
Changed!